### PR TITLE
CFD Simulations using the Jupyter Notebooks v1.0

### PR DESCRIPTION
**Pull request by moontrip**

### CODE REVIEW

**Review by meshl5555 — APPROVED** [review]

File: docs/kareen/usecase.md [hunk 1]
```diff
@@ -1,89 +1,94 @@
...
+<br/>
 
-Introductory Text.  High-level overview of product. A condimentum vitae sapien pellentesque habitant morbi tristique.
+## Description
 
-## Background 
+The Jupyter Notebooks have been widely used in science and engineering communities as an open-source interactive literate programming paradigm. It has the capabilities of combining text, images or videos with research codes, and can easily document, share, and reproduce data analysis. As for CFD, OpenFOAM is popularly used for computationally establishing wind effects on structures. To help beginners overcome the challenges of steep learning curve posed by OpenFOAM and provide users with the capabilities of generating repetitive jobs and advanced functions, this section presents the work to script the workflow for CFD simulations using OpenFOAM in the Jupyter Notebooks. The developed Jupyter Notebooks can aid in determining inflow conditions, creating mesh files for parameterized building geometries and running the selected solvers. This tool can also contribute to the education for CFD learning as online resources, which will be implemented in the [DesignSafe cyberinfrastructure (CI)](https://www.designsafe-ci.org).
```
Comment (kks32) [inline docs/kareen/usecase.md]: Could you please remove the Jupyter background info and talk about how to use Jupyter with OpenFOAM.

File: docs/kareen/usecase.md [hunk 2]
```diff
@@ -1,89 +1,94 @@
...
 
-* Please cite [AUTHORS et al. (20xx) - example of published project](https://doi.org/10.17603/ds2-3zdj-493) to acknowledge the use of any resources from this use case.
+* Please cite Ding and Kareem (2021) to acknowledge the use of any resources from this use case.<br>
+Ding, F., and Kareem, A. (2021). Using the Jupyter Notebooks as a tool for CFD simulations, 6th American Association for Wind Engineering (AAWE) Workshop (online),
+Clemson University, Clemson, SC, USA, May 12-14. [PDF](https://tigerprints.clemson.edu/cgi/viewcontent.cgi?article=1025&context=aawe)
 
-* Please cite [Rathje et al. (2017)](https://doi.org/10.1061/(ASCE)NH.1527-6996.0000246) to acknowledge the use of DesignSafe resources.  
+<br/>
 
-* This software is distributed under the GNU General Public License (https://www.gnu.org/licenses/gpl-3.0.html).
```
Comment (kks32) [inline docs/kareen/usecase.md:15]: We need the license info

Comment (kks32) [inline docs/kareen/usecase.md:15]: * This software is distributed under the [GNU General Public License](https://www.gnu.org/licenses/gpl-3.0.html).

File: docs/kareen/usecase.md [hunk 3]
```diff
@@ -1,89 +1,94 @@
...
-### Citation and Licensing
+### Citation
 
-* Please cite [AUTHORS et al. (20xx) - example of published project](https://doi.org/10.17603/ds2-3zdj-493) to acknowledge the use of any resources from this use case.
+* Please cite Ding and Kareem (2021) to acknowledge the use of any resources from this use case.<br>
+Ding, F., and Kareem, A. (2021). Using the Jupyter Notebooks as a tool for CFD simulations, 6th American Association for Wind Engineering (AAWE) Workshop (online),
+Clemson University, Clemson, SC, USA, May 12-14. [PDF](https://tigerprints.clemson.edu/cgi/viewcontent.cgi?article=1025&context=aawe)
 
-* Please cite [Rathje et al. (2017)](https://doi.org/10.1061/(ASCE)NH.1527-6996.0000246) to acknowledge the use of DesignSafe resources.  
+<br/>
```
Comment (kks32) [inline docs/kareen/usecase.md]:
```suggestion

```

File: docs/kareen/usecase.md [hunk 4]
```diff
@@ -1,89 +1,94 @@
...
 
-### Citation and Licensing
+### Citation
 
-* Please cite [AUTHORS et al. (20xx) - example of published project](https://doi.org/10.17603/ds2-3zdj-493) to acknowledge the use of any resources from this use case.
+* Please cite Ding and Kareem (2021) to acknowledge the use of any resources from this use case.<br>
+Ding, F., and Kareem, A. (2021). Using the Jupyter Notebooks as a tool for CFD simulations, 6th American Association for Wind Engineering (AAWE) Workshop (online),
+Clemson University, Clemson, SC, USA, May 12-14. [PDF](https://tigerprints.clemson.edu/cgi/viewcontent.cgi?article=1025&context=aawe)
 
-* Please cite [Rathje et al. (2017)](https://doi.org/10.1061/(ASCE)NH.1527-6996.0000246) to acknowledge the use of DesignSafe resources.
```
Comment (kks32) [inline docs/kareen/usecase.md:13]: * Please cite [Rathje et al. (2017)](https://doi.org/10.1061/(ASCE)NH.1527-6996.0000246) to acknowledge the use of DesignSafe resources.

File: docs/kareen/usecase.md [hunk 5]
```diff
@@ -1,89 +1,94 @@
...
 
-### Citation and Licensing
+### Citation
 
-* Please cite [AUTHORS et al. (20xx) - example of published project](https://doi.org/10.17603/ds2-3zdj-493) to acknowledge the use of any resources from this use case.
+* Please cite Ding and Kareem (2021) to acknowledge the use of any resources from this use case.<br>
```
Comment (kks32) [inline docs/kareen/usecase.md]:
```suggestion
* Please cite Ding and Kareem (2021) to acknowledge the use of any resources from this use case.
```

File: docs/kareen/usecase.md [hunk 6]
```diff
@@ -1,89 +1,94 @@
...
+Ding, F., and Kareem, A. (2021). Using the Jupyter Notebooks as a tool for CFD simulations, 6th American Association for Wind Engineering (AAWE) Workshop (online),
```
Comment (kks32) [inline docs/kareen/usecase.md]:
```suggestion
* Ding, F., and Kareem, A. (2021). Using the Jupyter Notebooks as a tool for CFD simulations, 6th American Association for Wind Engineering (AAWE) Workshop (online),
```

File: docs/kareen/usecase.md [hunk 7]
```diff
@@ -1,89 +1,94 @@
...
 
-[Link Example - this goes to Google](https://www.google.com)
+To address this issue in part, Jupyter Notebooks as an open-source project is developed to directly execute the functions for mesh generation, inflow settings, etc. in a user-friendly and interactive coding environment, aiming at alleviating the sophistication in running CFD models in OpenFOAM.
 
-## Header 2
+<br/>
```
Comment (kks32) [inline docs/kareen/usecase.md]: Please remove this section as it doesn't help explain the usecase itself.

**Review by kks32 — CHANGES_REQUESTED** [review]
The current usecase reads more like a paper and describes the motivation behind the use of Jupyter. We would like the usecase to be a demonstration / an example of how to use your code for others to reuse. 

We really like fig 1. Following on Fig 1, if you could write a section on Geometry, Mesh, Case Setup - Solver and Post processing.

**Comment by moontrip** [comment]
@erathje71 @kks32 We have revised our use case document following your suggestions and comments in this PR. We think that this is much improved from the last doc owing to your advices and we thank for your great efforts to enhance the quality of our use case.

Please feel free to leave comments in this PR if further changes are needed. 

Happy Easter!

DK

**Review by kks32 — APPROVED** [review]

**Pull request merged**